source/community/reactnative/package.json

```diff
@@ -43,16 +43,16 @@
     "jest": "^24.9.0",
     "metro-react-native-babel-preset": "^0.58.0",
     "react": "16.11.0",
-    "react-native": "0.62.0",
+    "react-native": "^0.62.2",
     "react-test-renderer": "16.11.0",
-    "rnpm-plugin-windows": "^0.2.11"
+    "rnpm-plugin-windows": "^0.2.13"
```

Comment:
Seems there is one moderate vulnerability in the dependency of this pkg. @IbrahimSulai can you check once ?
![image](https://user-images.githubusercontent.com/462275/100985323-db321080-3571-11eb-9de2-fe88e28445f7.png)

Reply:
This moderate is not getting solved, when doing both "npm audit fix" and "npm audit force fix".

We will take time and resolve this vulnerability in the upcoming days.